Add temporary info on how to re-enable chat (#343)
mastercoms/tf2cfg: docs/next_steps/troubleshoot.md

The command will only run on the first time you spawn in a match.

> ## Last update disabled my chat

Reviewer: Make this more generic: "My chat is disabled"

Author: done

> One of [recent TF2 updates](https://www.teamfortress.com/post.php?id=62459) introduced new commands to disable chat. mastercomfig was updated shortly after to accomodate to this change. In the result, the Very Low preset disables chat completely. If you are using Very Low preset or `messages=off` module, you can fix that by adding `messages=on` to your `modules.cfg` file.

Author: already fixed bro, saw that before you

> One of [recent TF2 updates](https://www.teamfortress.com/post.php?id=62459) introduced new commands to disable chat. mastercomfig was updated shortly after to accommodate to this change. In the result, the Very Low preset disables chat completely. If you are using Very Low preset or `messages=off` module, you can fix that by adding `messages=on` to your `modules.cfg` file. To disable chat the old way you can use `hud_saytext_time 0` command.

Reviewer: ```suggestion
[A TF2 update](https://www.teamfortress.com/post.php?id=62459) introduced the ability to completely disable text chat. mastercomfig was updated shortly after to accommodate this change, with the Very Low preset disabling chat completely. If you are using Very Low preset or the `messages=off` module, you can re-enable chat by adding `messages=on` to your `modules.cfg` file. To disable chat the old way you can use the `hud_saytext_time 0` command.
```

Author: Oh this is actually better. I was thinking only in scope of the recent update. And that it would be temporary.

Author: done
